src/components/User/Badges/Badge.css

```diff
@@ -0,0 +1,41 @@
+.Badge {
+  z-index: 1;
+  /*width: 32px;*/
+  /*height: 32px;*/
+}
+
+.Badge__Icon {
+  border-radius: 50%;
+  background-size: 192px 192px;
+  background-repeat: no-repeat;
+}
+
+.Badge__Primary {
```

Comment:
As these are variants to Badge__Icon I'd rename this to `Badge__Icon--primary`, `Badge__Icon--filled-mono`, etc.